test/built-ins/Temporal/Instant/prototype/round/accepts-plural-units.js

```diff
@@ -0,0 +1,23 @@
+// Copyright (C) 2021 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-temporal.instant.prototype.round
+description: round() accepts plural units.
+includes: [temporalHelpers.js]
+features: [Temporal]
+---*/
+
+const inst = Temporal.Instant.from("1976-11-18T14:23:30.123456789Z");
```

Comment:
```suggestion
const inst = new Temporal.Instant(1_000_000_000_123_456_789n);
```
(I picked this value to avoid testing the unrelated path through string parsing code, but also would be rounded by each of these rounding units)